android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,8 +1,8 @@
-• New OpenStreetMap data as of January 21
-• Dispay a speed limit sign in the navigation mode
-• Added Serbian (Cyrillic) and Latvian languages
-• Android Auto: sort bookmarks, improved location permission request experience
-• Added tower POIs
-• Added a setting for Kayak.com Hotel Links and a first-use opt-out option
+• New OpenStreetMap data as of February 27
+• Ability to manually arrange intermediate route points
+• Share a single selected track from the list
+• Improved routing for bicycles
+• Display a system notification when downloading maps and don't interrupt background downloads
+• Added a flashing Track Recording widget
```

Comment:
Sounds confusing

Reply:
please suggest a better wording :)

Reply:
how about "Added a Track Recording indicator on the main screen"